taskmanager/features/bootstrap/FeatureContext.php

```diff
@@ -1,31 +1,31 @@
 <?php
 
-use App\Application\Task\Status;
+use App\Domain\Process\Task\Status;
```

Comment:
of course there is a lot of naming convetionts and patterns. My recommendation is:
```text
Domain\{BoundedContext}\ # <- domain layer, ports
{Application|UseCases}\{BoundedContext}\ # <- application layers (CQRS) uses ports 
Infrastructure\{BoundedContext} # <- infra layers / adapters, implements ports
```

above concept is similar to Clean Architecture but I'd say that the most important is bounded context separation

Reply:
Yes, exactly! In all repository that I tried to interpret naming conventions is very different. Thanks for tip.